itests/eth_balance_test.go

```diff
@@ -69,3 +71,23 @@ func TestEthGetBalanceExistentIDMaskedAddr(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, ebal, ethtypes.EthBigInt{Int: balance.Int})
 }
+
+func TestEthGetBalanceBuiltinActor(t *testing.T) {
+	blockTime := 100 * time.Millisecond
+	client, _, ens := kit.EnsembleMinimal(t, kit.MockProofs(), kit.ThroughRPC())
+	ens.InterconnectAll().BeginMining(blockTime)
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+	defer cancel()
+
+	// Address for market actor
+	fid, err := address.NewFromString("f05")
+	require.NoError(t, err)
+
+	ethAddr, err := ethtypes.EthAddressFromFilecoinAddress(fid)
+	require.NoError(t, err)
+
+	ebal, err := client.EthGetBalance(ctx, ethAddr, "latest")
+	require.NoError(t, err)
+	require.Equal(t, ethtypes.EthBigIntZero, ebal)
```

Comment:
Hmm, I'd rather this be non-zero -- can we fund the market actor (just send it 10 FIL), and assert that's what the call returns?

Reply:
Nope!